templates/mainapp/404.html

```diff
@@ -0,0 +1,151 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bootstrap CSS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+
+	<title>404 page not found</title>
+</head>
+<style>
+	body {
+		background-color: #f9ce84;
+		color: #d7722c;
+	}
+
+	#mygif img {
+		margin-top: 5vh;
+		height: 80vh;
+		width: 80vh;
+	}
+
+	.page_404 {
+		padding: 10vh 0;
+	}
+
+	.four_zero_four_bg h1 {
+		font-size: 18rem;
+		font-family: "Pacifico", cursive;
+		margin-bottom: 8vh;
+		font-weight: 900;
+	}
+
+	.four_zero_four_bg h3 {
+		font-size: 80px;
+		font-family: "Sora", sans-serif;
+		font-weight: 900;
+	}
+
+	.link_404 {
+		color: #fff !important;
+		padding: 10px 20px;
+		background: #74603d;
+		margin: 20px 0;
+		border-radius: 20px;
+		border-bottom: 4px solid #eaaa3c;
+		font-family: "Sora", sans-serif;
+		display: inline-block;
+		outline: none;
+		font-weight: 900;
+	}
+
+	.contant_box_404 {
+		margin-top: -50px;
+	}
+
+	a {
+		text-decoration: none;
+		font-family: "Sora", sans-serif;
+	}
+
+	p {
+		font-weight: 800;
+		font-family: "Sora", sans-serif;
+	}
+
+	.h2,
+	h2 {
+		font-weight: 900;
+		font-family: "Sora", sans-serif;
+	}
+
+	@media screen and (max-width: 992px) {
+		.four_zero_four_bg h1 {
+			font-size: 9rem;
+			margin-bottom: 5vh;
+		}
+
+		#mygif img {
+			margin-top: 1vh;
+			height: 50vh;
+			width: 50vh;
+			margin-left: 5vw;
+		}
+
+		.page_404 {
+			width: 97vw;
+			padding: 0vh 0;
+		}
+
+		.link_404 {
+			margin: 1px 0;
+		}
+	}
+</style>
+
+<body>
+	<div class="container-fluid">
+
+		<div class="row">
+			<div class="col col-lg-6 col-12 ">
+				<div id="mygif">
+					<img src="https://media.giphy.com/media/1bHTqyppyN99JwV5FW/source.gif">
```

Comment:
Store the gif [here ](https://github.com/Praful932/Kitabe/tree/master/static/mainapp/gif)instead of specifying as a url

Reply:
done